packages/react-topology/src/Visualization.ts

```diff
@@ -45,12 +45,20 @@ export class Visualization extends Stateful implements Controller {
   }
 
   @action
-  fromModel(model: Model): void {
+  fromModel(model: Model, merge: boolean = true): void {
     const oldGraph = this.graph;
 
+    // If not merging, clear out the old elements
+    if (!merge) {
+      this.elements = {};
+    }
```

Comment:
We may want to go through the routine of destroying the element for the sake of consistency.